_posts/2024-06-06-opensearch-partnering-with-datastax-on-generative-ai.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Announcing an OpenSearch and DataStax partnership on Generative AI"
+authors:
+  - zznate 
+date: 2024-06-06
+categories:
+  - community
+  - partners
+meta_keywords: 
```

Comment:
Please update the meta with the following: 

**meta_keywords:** Generative AI, retrieval augmented generation , DataStax HCDP, OpenSearch integrations

**meta_description:** Learn about the collaboration between open source startup DataStax and the OpenSearch Project on integration efforts to support Generative AI developers.